xml/System.Timers/Timer.xml

```diff
@@ -345,7 +345,7 @@
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- If the <xref:System.Timers.Timer> is already enabled when the <xref:System.Timers.Timer.Start%2A> method is called, the interval is reset. If <xref:System.Timers.Timer.AutoReset%2A> is `false`, the <xref:System.Timers.Timer.Start%2A> method must be called in order to start the count again.
+ If <xref:System.Timers.Timer.AutoReset%2A> is `false`, the <xref:System.Timers.Timer.Start%2A> method must be called in order to start the count again.
```

Comment:
There is no System.Timers.Timer.Start, so, yeah :)

Reply:
@stephentoub 
> There is no System.Timers.Timer.Start, so, yeah :)

There is: https://source.dot.net/#System.ComponentModel.TypeConverter/System/Timers/Timer.cs,d8d4c6fbc471f649
But it will not reset interval on consecutive calls:
https://source.dot.net/System.ComponentModel.TypeConverter/R/c458cff06715bb66.html
```cs
public bool Enabled
{
    get => _enabled;
    set
    {
        if (DesignMode)
        {
            _delayedEnable = value;
            _enabled = value;
        }
        else if (_initializing)
        {
            _delayedEnable = value;
        }
        else if (_enabled != value)
        {
            if (!value)
            {
                if (_timer != null)
                {
                    _cookie = null;
                    _timer.Dispose();
                    _timer = null;
                }
                _enabled = value;
            }
            else
            {
                _enabled = value;
                if (_timer == null)
                {
                    ObjectDisposedException.ThrowIf(_disposed, this);

                    int i = (int)Math.Ceiling(_interval);
                    _cookie = new object();
                    _timer = new Threading.Timer(_callback, _cookie, Timeout.Infinite, Timeout.Infinite);
                    _timer.Change(i, _autoReset ? i : Timeout.Infinite);
                }
                else
                {
                    UpdateTimer();
                }
            }
        }
    }
}
```

Reply:
Huh, you're right. I only ever used Enabled directly and could not have told you there was a one-line helper on top of it :-)